examples/helloworld/mock_helloworld/hw_mock.go

```diff
@@ -1,4 +1,4 @@
-// Automatically generated by MockGen. DO NOT EDIT!
```

Comment:
Revert (or upgrade gomock version?)?

Reply:
No, we need this.
`golint` ignores files with
```
// Code generated ... DO NOT EDIT.
```
https://github.com/golang/lint/blob/738671d3881b9731cc63024d5d88cf28db875626/lint.go#L119

Mock's generated code doesn't comply with it.

Reply:
Oh, so it is hand-edited?  I guess that's OK?